test/jtreg/generator/test8257892/LibUnsupportedTest.java

```diff
@@ -91,16 +91,19 @@ public void testFieldTypes() {
     }
 
     @Test
-    public void testIgnoredMethods() {
+    public void testProblematicMethods() {
```

Comment:
In this case we were checking that some methods were NOT generated, even though we could perfectly compute their layouts.